src/Candidates/Client.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace Elective\ApiClients\Candidates;
+
+use Elective\ApiClients\ApiClient;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+use Symfony\Component\HttpFoundation\RequestStack;
+
+/**
+ * Elective\ApiClients\Candidates\Client
+ *
+ * @author Sammy Ha
+ */
+class Client extends ApiClient
+{
+    public const ACTION_VIEW        = 'view';
```

Comment:
so, these look duplicated everywhere. In that case move them to one common class. A good place for this is the ApiClient as every other client extends it. In a nutshell if you see repeated code, move it to one common class